apps/foundation-demo/src/app/thank-you/components/print-order-summary/print-order-summary.component.scss

```diff
@@ -0,0 +1,5 @@
+@import 'daff-util';
```

Comment:
We don't need this file.

Reply:
I added a comment to this pull request, since it makes more sense to go there: https://github.com/graycoreio/daffodil/pull/186#pullrequestreview-192144211